<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CTcpEdgeTestDataView_ModelDevRead</class>
 <widget class="QWidget" name="CTcpEdgeTestDataView_ModelDevRead">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CTcpEdgeTestDataView_TimeReadReq</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QComboBox" name="_ModelAll_">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>模型名数组(Models)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbbModels">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnAdd">
        <property name="text">
         <string>添加</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/Resources/add.png</normaloff>:/Resources/add.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDelete">
        <property name="text">
         <string>删除</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/Resources/delete.png</normaloff>:/Resources/delete.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QCheckBox" name="__ModelAll__">
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>模型设备查询范围(ModelAll)</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QTableWidget" name="_Models_">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>200</height>
      </size>
     </property>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QCheckBox" name="__Models__">
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../../../TcpEdgeServer/TcpEdgeServer.qrc"/>
 </resources>
 <connections/>
</ui>
